src/service.js

```diff
@@ -1,22 +1,30 @@
 let express = require('express');
 let http = require('http');
+let session = require('./session');
+let adb = require('./adb');
 
 let server;
 
-exports.start = function start(port = 8080) {
+exports.start = function start(options = {}) {
   let app = express();
+
+  app.use(session());
+  app.use(adb({ path: options.adbPath }));
+
   app.get('/', (req, res) => res.send('200 OK'));
   app.get('/crashes', require('./list_crash_reports'));
   app.get('/crashes/:id', require('./get_crash_report'));
-  app.get('/info', require('./info'));
+  app.get('/devices', require('./devices'));
+  app.get('/device', require('./device'));
```

Comment:
Weren't we thinking that you could do `/device/<device id>` (perhaps in addition to the header mechanism)?